src/main/java/com/teragrep/rlp_03/SocketPoll.java

```diff
@@ -77,10 +77,12 @@ public void poll() throws IOException {
                         connectionContext.handleEvent(selectionKey);
                     }
                     catch (CancelledKeyException cke) {
+                        LOGGER.error("SocketPoll.poll CancelledKeyException caught: " + cke.getMessage());
```

Comment:
string concat will not do, see Procedure: Variable Logging

Reply:
fix: f9a8f1327ed4cc2aca5059ac10a03039692a026c